ozark/pom.xml

```diff
@@ -7,12 +7,53 @@
         <artifactId>parent</artifactId>
         <version>1.0.0-m03-SNAPSHOT</version>
     </parent>
-    <artifactId>ozark</artifactId>
-    <packaging>jar</packaging>
-    <name>Ozark ${project.version} MVC RI</name>
+    <artifactId>javax.mvc</artifactId>
```

Comment:
Hmm, I've seen that the all RIs uses this convention, for example look at Json-P RI https://java.net/projects/jsonp/sources/git/content/impl/pom.xml?rev=6bac828463f2b714effad5769d6f0a2bfc420af2